### PR TITLE
Addresses #729: Dropdown menu has sufficient space to right side

### PR DESCRIPTION
**Pull request by harshit98**
Fixes issue #729 

Changes: 
- Given dropdown menu sufficient space to the right side.
- Now it won't be get collapsed with the scroll bar.

Demo Link: https://harshit98.github.io/susper.com/

Screenshots for the change: 

screenshot from 2017-08-12 10-48-10

Please review @nikhilrayaprolu @Marauderer97

### CODE REVIEW

File: src/app/dropdown/dropdown.component.css [hunk 1]
```diff
@@ -131,6 +131,12 @@ hr {
     padding-top: 10px;
 }
 
+@media screen and (max-width: 2560px) {
```
Comment (Marauderer97) [inline src/app/dropdown/dropdown.component.css]: donot put a max width on this. Change this to min width form. Let us extend it to even beyond 2560px.
At 2670px:
image

File: src/app/dropdown/dropdown.component.css [hunk 2]
```diff
@@ -131,6 +131,12 @@ hr {
...
+    .dropdown-menu-box {
```
Comment (Marauderer97) [inline src/app/dropdown/dropdown.component.css]: @harshit98 Why is this happening? This was fixed by me in a previous PR. I think this is a result of #726. Please look into this in this PR and fix it.
image

File: src/app/dropdown/dropdown.component.css (hunk 1 above)
Comment (harshit98) [inline src/app/dropdown/dropdown.component.css]: Okay working on it.

File: src/app/dropdown/dropdown.component.css (hunk 2 above)
Comment (harshit98) [inline src/app/dropdown/dropdown.component.css]: I will look into this issue.

**Comment by harshit98** [comment]
Done @Marauderer97. Please review.

File: src/app/dropdown/dropdown.component.css [hunk 3]
```diff
@@ -131,6 +131,24 @@ hr {
     padding-top: 10px;
 }
 
+@media screen and (max-width: 2670px) {
+    .dropdown-menu-box {
+        margin-left: -80%;
+    }
+}
+
```
Comment (Marauderer97) [inline src/app/dropdown/dropdown.component.css]: @harshit98 Simply change to min-width 817 px (or from whichever size you want the change to be applied). This is redundant.

**Comment by harshit98** [comment]
Done @Marauderer97. Please review.

**Comment by Marauderer97** [comment]
@harshit98 please look at the changes in the code i have requested

**Comment by harshit98** [comment]
@Marauderer97 Actually if I use `min-width` from where I want to apply the changes, it is reverting back the position of dropdown. This is currently working fine.

**Comment by Marauderer97** [comment]
Have you tried min-width from 817px?

**Comment by harshit98** [comment]
Yes @Marauderer97

**Comment by Marauderer97** [comment]
@harshit98 Please find out why this is happening, it shouldn't happen. Like you said we are in the last phase of GSoC and maintaining code quality is equally important. Especially since the CSS was previously very messy and we don't want it to revert it to the same state

**Comment by harshit98** [comment]
@Marauderer97 Done! :smile:

**Comment by Marauderer97** [comment]
It seems to be working fine, though I think the CSS can still be improved and shortened. However it can be approved for all practical reasons :+1:  @nikhilrayaprolu I leave it to your discretion

**Pull request merged**